src/getHumanAge.test.js

```diff
@@ -3,8 +3,20 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should return an array with two elements', () => {
```

Comment:
We are checking it in each case, no need to check separately that array should return an array with 2 elements